types/stripe-v3/stripe-v3-tests.ts

```diff
@@ -43,6 +43,9 @@ describe("Stripe elements", () => {
         card.on('ready', () => {
             console.log('ready');
         });
+        card.addEventListener('ready', () => {
```

Comment:
@puggan  could you please add this test as well

```javascript
        card. addEventListener('change', (response) => {
            if (response) {
                console.log(response.elementType, response.brand);
            }
        });
```

Reply:
Added